### PR TITLE
Add ElasticUI framework for client

### PR DESCRIPTION
**Pull request by bryophyta**
## What does this change?

- Install Elastic UI (EUI) and dependencies
- Replace existing client functionality with EUI components.

## Images

|Before|After|
|-|-|
|image|image|

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

**Comment by twrichards** [comment]
think search might be a bit broken 
image

File: newswires/app/db/FingerpostWireEntry.scala
[the remark this answers is not in the document]
Comment (bryophyta) [reply newswires/app/db/FingerpostWireEntry.scala:1]: accidental commit 😬

File: newswires/client/src/NavigationList.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+// 	</EuiFlexGroup>
+// 	<EuiText size="s" color="subdued">
+// 		<p>
+// 			The docked status is being stored in{' '}
+// 			<EuiCode>localStorage</EuiCode>.
+// 		</p>
+// 	</EuiText>
+// 	<NavigationList />
+// </div>
+// </EuiCollapsibleNav>
```
[the remark this answers is not in the document]
Comment (bryophyta) [reply newswires/client/src/NavigationList.tsx:146]: didn't mean to commit this here 👍

**Comment by bryophyta** [comment]
> think search might be a bit broken image



> think search might be a bit broken image

Thanks for this! I haven't been able to reproduce yet though..

When you were running locally, has the `pg_trgm` extension been enabled in the local db? That's a possible source of it breaking like this. If that's not it, are you able to take a look at the error that's being sent back from the server? (The JSON parsing issue will probably because Play's sending back HTML for an error page.)

**Comment by bryophyta** [comment]
@twrichards I've removed the comments & code that weren't meant to be committed -- thanks for catching those!

It all seems to be working okay for me locally and I haven't been able to reproduce the issue you were having with search. Perhaps we can pair tomorrow on trying to reproduce?

**Comment by bryophyta** [comment]
We're confident that the issue with search locally isn't a regression here. We're going to add a fix here, but in the short term if you run this in the newswires db container it should unblock locally: `psql -U postgres -d newswires -c "create extension if not exists pg_trgm;"`

**Pull request merged**